### PR TITLE
Fix admin for concierge 4.0 beta

### PR DESCRIPTION
**Pull request by Awarua-**

### CODE REVIEW

File: admin.js [hunk 1]
```diff
@@ -104,27 +104,22 @@
             }
             return false;
         };
-    },
-
-    helpHook = function (moduleName, origionalHelp, config) {
```
Comment (drkno) [inline admin.js]: So help messages are no longer filtered?

File: admin.js [hunk 2]
```diff
@@ -104,27 +104,22 @@
...
-                return origionalHelp.call(this, commandPrefix);
-            }
-            return false;
-        };
     };
 
 exports.load = function () {
-    cfg = exports.platform.config.getConfig('admin');
-    var loadedModules = exports.platform.modulesLoader.getLoadedModules();
+    cfg = this.config;
```
Comment (drkno) [inline admin.js]: `exports.config` is safer if the this context cannot be bound

**Review by drkno — CHANGES_REQUESTED** [review]

File: admin.js (hunk 1 above)
Comment (Awarua-) [reply admin.js]: I don't remember why this is, but probably a good reason

File: admin.js (hunk 2 above)
Comment (Awarua-) [reply admin.js]: Changed

File: admin.js (hunk 1 above)
Comment (Awarua-) [reply admin.js]: Added back

**Pull request merged**